.travis.yml

```diff
@@ -10,6 +10,7 @@ python:
 cache: pip
 
 before_install:
+  - sudo add-apt-repository -y ppa:jonathonf/texlive-2016
```

Comment:
I mean it is *to be fixed* - not fixed by using this repo.

Should I revert ppa:jonathonf/texlive-2016?